config/schedule.rb

```diff
@@ -7,3 +7,8 @@
 every :day, at: '3:00 am', roles: [:job_creator] do
   rake 'clear_expired_sessions'
 end
+
+every Figaro.env.queue_health_check_frequency_seconds.to_i, roles: [:job_creator] do
+  runner 'WorkerHealthChecker.check'
+  runner 'WorkerHealthChecker.enqueue_dummy_jobs'
+end
```

Comment:
Figaro was not as standalone as I thought, just gonna require the whole rails app -- this should only happen once per deploy so I don't think it will slow things down too much